kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/richmessaging/AlRichMessageFactory.java

```diff
@@ -39,18 +40,18 @@ public static AlRichMessageFactory getInstance() {
 
     public AlRichMessage getRichMessage(Context context, LinearLayout containerView, Message message, ALRichMessageListener listener, AlCustomizationSettings alCustomizationSettings) {
         int type = -1;
-        if(message.getMetadata().containsKey("templateId"))
-        type = Integer.parseInt(message.getMetadata().get("templateId"));
+        if (message.getMetadata().containsKey("templateId"))
+            type = Integer.parseInt(message.getMetadata().get("templateId"));
```

Comment:
Add {

Reply:
is it for readibility?

Reply:
https://wiki.c2.com/?AlwaysUseBracesOnIfThen

Reply:
added braces

Reply:
will try to keep all conditional statements one liners in the future, seemed like a good readable approach. will reduce the need of braces.

Reply:
Still we should add { even if that's one line